packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/ccn/3043/indemnite.js

```diff
@@ -11,30 +11,34 @@ export function getSalaireRef({ salaireRefLegal }) {
 
 export function getIndemnite({ salaireRef, anciennete }) {
   let error;
-  let remainingAnciennete = anciennete;
   let indemniteConventionnelle = 0;
   let formulesConventionnelles = [];
+  const trancheAnciennete = {
+    a10: Math.max(0, round(anciennete - 10)),
+    a5: Math.min(5, Math.max(0, round(anciennete - 5))),
+    a2: Math.min(5, round(anciennete))
+  };
+
+  let inputConventionnels = {
+    "Salaire de référence (Sref)": round(salaireRef),
+    "Ancienneté entre 2 et 5ans (A1)": trancheAnciennete.a2,
+    "Ancienneté 6 ens 10ans (A2)": trancheAnciennete.a5,
+    "Ancienneté au dela de 10ans (A3)": trancheAnciennete.a10
+  };
 
-  if (remainingAnciennete > 10) {
-    indemniteConventionnelle +=
-      (1 / 5) * salaireRef * (remainingAnciennete - 10);
-    formulesConventionnelles.push(
-      `1/5 * ${round(salaireRef)} * (${remainingAnciennete} - 10)`
-    );
-    remainingAnciennete = 10;
+  if (trancheAnciennete.a10) {
+    indemniteConventionnelle += (1 / 5) * salaireRef * trancheAnciennete.a10;
+    formulesConventionnelles.push(`1/5 * Sref * ${trancheAnciennete.a10}`);
   }
-  if (remainingAnciennete > 5) {
-    indemniteConventionnelle = (1 / 6) * salaireRef * (remainingAnciennete - 5);
-    formulesConventionnelles.push(
-      `1/6 * ${round(salaireRef)} * (${remainingAnciennete} - 5)`
-    );
-    remainingAnciennete = 5;
+
+  if (trancheAnciennete.a5) {
+    indemniteConventionnelle += (1 / 6) * salaireRef * trancheAnciennete.a10;
+    formulesConventionnelles.push(`1/6 * Sref * ${trancheAnciennete.a5}`);
   }
-  if (remainingAnciennete > 2) {
-    indemniteConventionnelle = (1 / 10) * salaireRef * remainingAnciennete;
-    formulesConventionnelles.push(
-      ` + 1/10 * ${round(salaireRef)} * ${remainingAnciennete}`
-    );
+
+  if (trancheAnciennete.a2 > 2) {
+    indemniteConventionnelle += (1 / 10) * salaireRef * trancheAnciennete.a2;
+    formulesConventionnelles.push(`1/10 * Sref * ${trancheAnciennete.a2}`);
   } else {
     error =
       "Aucune indemnité de licenciement n'est prévue en deça de 2 ans d'ancienneté.";
```

Comment:
Il manque un accent à en deçà ^^' (c'est pas grave si tu le mets pas hein)

Reply:
je crois que je suis faché avé les accents ;)

Reply:
The french NIGHTMARE https://www.youtube.com/watch?v=5YO7Vg1ByA8